### PR TITLE
Create initial PyQt application

### PR DESCRIPTION
**Pull request by filxxip**

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "python.formatting.provider": "black"
```
Comment (jakubpoltoraczyk) [inline .vscode/settings.json:2]: To whole file
This should not be a part of PR, that's some kind of custom environment settings

File: PhoneApp [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+python3 main.py
```
Comment (jakubpoltoraczyk) [inline PhoneApp]: Add new line at the end of file

File: main.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+#in future to remove, now it is the first window of our application
+class Window(QDialog):
+    def __init__(self, master):
+        super().__init__()
+        label = QLabel(self)
+        label.setText("dsfds")
+
+app = QApplication(sys.argv)
+window = PhoneApp(app, "SPOTIFY", "style.css", Window, 1000, 800)
+sys.exit(app.exec())
```
Comment (jakubpoltoraczyk) [inline main.py]: Add new line at the end of file

File: main.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+from PyQt5.QtWidgets import QApplication, QDialog, QLabel
+import sys
+from src.basis_app_class import PhoneApp
+
+#in future to remove, now it is the first window of our application
```
Comment (jakubpoltoraczyk) [inline main.py]:
```
# todo: in future to remove, now it is the first window of our application
```
Mark each comment in this style as `todo:`

File: main.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+class Window(QDialog):
+    def __init__(self, master):
+        super().__init__()
+        label = QLabel(self)
+        label.setText("dsfds")
```
Comment (jakubpoltoraczyk) [inline main.py]: Please set some kind of normal text :)

File: src/basis_app_class.py [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+    master: QApplication = field(repr=False)
+    title: str
+    style_file: str
+    first_window: object
+    height: InitVar[int]
+    width: InitVar[int]
+    widget: QStackedWidget = field(init=False, repr=False)
+    _windows: dict = field(init=False, repr=False)
+
+    def __post_init__(self, screen_widht: int, screen_height: int) -> None:
```
Comment (jakubpoltoraczyk) [inline src/basis_app_class.py]: `screen_widht` -> `screen_width`

File: src/basis_app_class.py [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+        """Initialize a new PhoneApp instance
+
+        Params:
+            screen_widht (int): width of app
+            screen_height (int): height of app"""
+        self.widget = QStackedWidget()
+        self.master.setStyleSheet(open(f'src/{self.style_file}').read())
```
Comment (jakubpoltoraczyk) [inline src/basis_app_class.py]: I know that Python close files automatically, but please close this file :)
I think it's more clear and in a good pratice

**Review by jakubpoltoraczyk — APPROVED** [review]

**Comment by filxxip** [comment]
I've added some new improvements to this class, now i thing some parts are much more clean.

**Pull request merged**